client/card-readers/settings/sections/test/index.test.js

```diff
@@ -137,12 +137,6 @@ describe( 'Card Reader Branding Details section', () => {
 		render( <BrandingDetailsSection /> );
 
 		const logo = screen.queryAllByText( 'File Upload' );
-		expect( logo.length ).toEqual( 2 );
-
-		const primary = screen.getByText( 'Primary color' );
-		expect( primary ).toBeInTheDocument();
-
-		const secondary = screen.getByText( 'Secondary color' );
-		expect( secondary ).toBeInTheDocument();
+		expect( logo.length ).toEqual( 1 );
```

Comment:
Should this constant be named something like "fileUploadField" instead of "logo"? In essence, we are querying all file upload controls (fields) on the page rather than all logos ;)